docs/ocis/development/testing.md

```diff
@@ -203,16 +234,16 @@ If you want to work on a specific issue
 
 1. locally run each of the tests marked with that issue in the expected failures file.
 
-    E.g.:
+   E.g.:
 
-    ```bash
-    make test-acceptance-from-core-api \
-    TEST_SERVER_URL=https://localhost:9200 \
-    TEST_OCIS=true \
-    TEST_WITH_GRAPH_API=true \
-    STORAGE_DRIVER=OCIS \
-    BEHAT_FEATURE='tests/acceptance/features/coreApiVersions/fileVersions.feature:147'
-    ```
+   ```bash
+   make test-acceptance-from-core-api \
+   TEST_SERVER_URL=https://localhost:9200 \
```

Comment:
Should we do it something like this or hard coding is fine?
```suggestion
   TEST_SERVER_URL=<ocis-instance-url> \
```

Reply:
Most people run it locally so it should be fine but give me suggestions, I couldn't decide here

Reply:
Let it be as an example. I will write a note on `TEST_SERVER_URL` envvar

Reply:
Added here https://github.com/owncloud/ocis/blob/82e1423056d8d214a57b57a5ae2285561659a8e6/docs/ocis/development/testing.md?plain=1#L95-L97